game-headless/src/test/java/org/triplea/game/server/debug/ChatHandlerTest.java

```diff
@@ -1,91 +0,0 @@
-package org.triplea.game.server.debug;
-
-import static org.mockito.AdditionalMatchers.not;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.ArgumentMatchers.endsWith;
-import static org.mockito.Mockito.doAnswer;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-
-import java.util.function.Consumer;
-import java.util.logging.Level;
-import java.util.logging.LogRecord;
-import org.junit.jupiter.api.Nested;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
-
-final class ChatHandlerTest {
-  @ExtendWith(MockitoExtension.class)
-  @Nested
-  final class PublishTest {
-    private final ChatHandler chatHandler = new ChatHandler();
-    @Mock private Consumer<String> sendChatMessage;
-
-    private LogRecord newLoggableLogRecord() {
-      return new LogRecord(Level.WARNING, "message");
-    }
-
-    private LogRecord newLoggableLogRecordWithMultipleLines() {
-      return new LogRecord(Level.WARNING, "message\nmessage2\n");
-    }
-
-    private LogRecord newUnloggableLogRecord() {
-      return new LogRecord(Level.FINEST, "message");
-    }
-
-    private void publish(final LogRecord record) {
-      synchronized (chatHandler) {
-        chatHandler.publish(record, sendChatMessage);
-      }
-    }
-
-    @Test
-    void shouldSendChatMessageWhenRecordIsLoggable() {
```

Comment:
@trevan FWIW, this test written by Ssoloff I think is something of a good test class. Just wanted to refer you to this example.